Composer/packages/client/src/pages/publish/publishPageUtils.tsx

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ApiStatus } from '../../utils/publishStatusPollingUpdater';
+
+import { Bot, BotStatus, BotPublishHistory, BotPropertyType } from './type';
+
+export const deleteNotificationInterval = 5000;
+
+export const generateComputedData = (botProjectData) => {
```

Comment:
what's the type of botProjectData? please add.

Reply:
Add type